patches/@mauron85+react-native-background-geolocation+0.6.3.patch

```diff
@@ -532,3 +532,73 @@ index ad66b5c..0000000
 -
 -}
 \ No newline at end of file
+<<<<<<< HEAD
```

Comment:
I cant tell if this is a merge conflict or part of the patch, but ether way this is the jetifier changes so it's not needed, rever.